notebook.ipynb

```diff
@@ -2,13 +2,12 @@
  "cells": [
```

Comment:
<p>This histogram does not look right. The age should be a numeric variable and in this data set it is a string. You can either: find out how to deal with ranges like this or just remove this histogram.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nasheya98/final-project/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>